src/app.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+/*
+The file is concerned about maintaining the angular
+part of the app
+*/
+let invApp = angular.module("invertedIndex", []);
+
+
+invApp.controller("invertedController", ["$scope", function($scope) {
+  let invIndex = new Index();
+  $scope.uploadedFiles = {};
+  $scope.allFlag = false;
+  $scope.allFilesIndexed = {};
+  /**
+  $scope.getIndex is executed when the getIndex button is clicked
+  */
+  $scope.getIndex = function() {
+    let fileChoice = $scope.uploadSelected;
+    if(fileChoice === undefined) {
+      alert("Select a file to get index");
+      return false;
+    }
+    //if index was created for that file
+    if(invIndex.createIndex($scope.uploadedFiles[fileChoice].text, fileChoice)) {
+      //gets the indexed words
+      let indexes = invIndex.getIndex(fileChoice);
+      $scope.indexDisplay = true;
+      $scope.indexed = [
+        {
+          indexes: indexes,
+          documents: invIndex.getDocuments(fileChoice),
+          indexedFile: fileChoice
+        }
+      ];
+      //keeps track of files that have been indexed
+      $scope.allFilesIndexed[fileChoice] = true;
+
+      //to check if two files or above have been indexed
+      //so that an option to search all files can be added
+      if(Object.keys($scope.allFilesIndexed).length > 1) {
```

Comment:
**Code quality** -- No need to check length > 1. if(obj.length) returns true if it is >= 1.